.rubocop.yml

```diff
@@ -1,6 +1,9 @@
 AllCops:
   SuggestExtensions: false
 
+Style/ArgumentsForwarding:
```

Comment:
I can be wrong, but could you please try to move `Style/ArgumentsForwarding` below all other instructions in this file?

Reply:
As they can overwrite the instructions above, including `Style/ArgumentsForwarding`

Reply:
It didn't help. I tried. :)

Reply:
@Rukomoynikov It's very odd. I don't quite understand why these rules are being ignored in `.rubocop.yml` or `.standard.yml`.